src/common/misc/ListElementListModel.ts

```diff
@@ -4,11 +4,16 @@ import { OperationType } from "../api/common/TutanotaConstants"
 import Stream from "mithril/stream"
 import { ListLoadingState, ListState } from "../gui/base/List"
 
-export type ListElementListModelConfig<ElementType> = Omit<ListModelConfig<ElementType, Id>, "getItemId" | "isSameId">
+export type ListElementListModelConfig<ElementType> = Omit<ListModelConfig<ElementType, Id>, "getItemId" | "isSameId"> & {
```

Comment:
If you need to both add and remove something maybe it makes sense to just mention everything.
You can still reference types of fields from `ListModelConfig`.

So in my head it would be like

```ts
export interface ListElementListModelConfig<ElementType> = {
  fetch: ListModelConfig<ElementType, Id>["fetch"],
  //...
}
```

but I'm not sure if it's easier to read actually